src/page-multi-facility/CreateBaselineScenarioPage.tsx

```diff
@@ -57,7 +56,6 @@ const CreateBaselineScenarioPage: React.FC = () => {
 
     if (scenario) {
       dispatchScenarioUpdate(scenario);
-      navigate("/facility");
```

Comment:
We no longer send new users to the create facility page but instead send them to the baseline scenario page and present them with a reference data modal to add new facilities.